package.json

```diff
@@ -16,7 +16,7 @@
     "lint:lib": "eslint lib",
     "test": "npm-run-all lint test:all",
     "test:all": "npm-run-all test:unit",
-    "test:unit": "multi-tape lib/**/*.unit.spec.js lib/**/**/*.unit.spec.js lib/middleware/**/*.unit.spec.js lib/page/**/*.unit.spec.js lib/**/**/**/*.unit.spec.js"
+    "test:unit": "USER_DATASTORE_URL='' USER_FILESTORE_URL='' SUBMITTER_URL='' SERVICE_SECRET='' SERVICE_TOKEN='' SERVICE_SLUG='' multi-tape lib/**/*.unit.spec.js lib/**/**/*.unit.spec.js lib/middleware/**/*.unit.spec.js lib/page/**/*.unit.spec.js lib/**/**/**/*.unit.spec.js"
```

Comment:
Could this cause probs when running tests individually? e.g. when working on a test I normally do `./node_modules/.bin/tape lib/client/metrics/metrics.unit.spec.js` - maybe the `test.only()` is better though

Reply:
Caveat emptor/tester

If you're running the tests individually you'll see the error clearly for that one file and it's up to you for your ENV to be in the correct state ;)

Reply:
NB. this was occasionally biting me and taking me too long for the penny to drop that I thought it sensible to enforce it